metagpt/actions/research.py

```diff
@@ -110,7 +111,7 @@ async def run(
         system_text = system_text if system_text else RESEARCH_TOPIC_SYSTEM.format(topic=topic)
         keywords = await self._aask(SEARCH_TOPIC_PROMPT, [system_text])
         try:
-            keywords = json.loads(keywords)
+            keywords = OutputParser.extract_struct(keywords, "list")
```

Comment:
should we use list instead of "list"?

Reply:
Yes, it's more elegant. It has already been updated.